### DISCUSSION TITLE
自定义高亮颜色

### DISCUSSION THREAD

**Discussion by learnMyCode** [vuejs/language-tools#4028] [Q&A]
如题，怎么配置sfc中的高亮颜色呢

**Comment by angelhdzmultimedia** [comment]
This is a good question. I've played with VSCode's Token Customizations TextMate rules to for example modify the color of the `this` (JS/TS) and `self` (Python) keywords. But I have researched to modify Vue .sfc tokens and haven't found a way.

Subscribing for a solution.
